app/controllers/application_controller.rb

```diff
@@ -7,10 +7,10 @@ class ApplicationController < ActionController::Base
   protected
 
   def configure_permitted_parameters
-    devise_parameter_sanitizer.permit(:sign_up, keys: [:name])
+    devise_parameter_sanitizer.permit :sign_up, keys: [:name]
```

Comment:
Ok nhé, chị check lại rồi :D
`keys: - An Array of keys that also should be permitted`